### ISSUE TITLE
NTR: X-ray or CT: squamosal suture craniosynostosis (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9000]
New term request X-ray or CT: squamosal suture craniosynostosis (elthat_140730081137): Craniostenosis; Squamous; between the temporal and parietal bones

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000612
- label: Squamosal suture synostosis
- parents: [Craniosynostosis (HP:0001363)](https://hpo.jax.org/app/browse/term/HP:0001363)
- pmid: PMID:27438438
- def: Premature closure of the squamosal suture, which is one of the lateral minor skull sutures, separating the parietal and squamous temporal bones.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9000
- synonyms:
